src/chart/Summary.js

```diff
@@ -44,11 +48,15 @@
                         context.render();
                     }
                 }, _);
-            }
+            }       
         }
         return retVal;
     };
 
+    Summary.prototype.summary_format = function() {
```

Comment:
@mlzummo , I tried it in a different way and I forgot to remove it. I will make those changes and commit it again. Sorry about that.